src/test/script_tests.cpp

```diff
@@ -1840,8 +1862,76 @@ BOOST_AUTO_TEST_CASE(formatscriptflags)
     // quick check that FormatScriptFlags reports any unknown/unexpected bits
     BOOST_CHECK_EQUAL(FormatScriptFlags(SCRIPT_VERIFY_P2SH), "P2SH");
     BOOST_CHECK_EQUAL(FormatScriptFlags(SCRIPT_VERIFY_P2SH | (1u<<31)), "P2SH,0x80000000");
-    BOOST_CHECK_EQUAL(FormatScriptFlags(SCRIPT_VERIFY_TAPROOT | (1u<<27)), "TAPROOT,0x08000000");
```

Comment:
Note for reviewers this change is needed b/c (1u<<26) and (1u<<27) are no longer `unknown/unexpected bits` . They are used for `SCRIPT_VERIFY_OP_CAT` and `SCRIPT_VERIFY_DISCOURAGE_OP_CAT`. Perhaps a comment here could help the next person to get stuck here .

@EthanHeilman